tasks/IWantThis.tex

```diff
@@ -0,0 +1,91 @@
+\section{I Want This [Party Host]}
+A guest at the party speaks English, but with only a limited vocabulary. They want a drink, an hors d'oeuvre, and a set of utensils, but do not know the words to describe them. As such, they will ask the robot for one through gesturing, and will ask for recommendations made by the robot through gestures. They will also discuss a picture on the wall, which the robot will determine by analyzing their gaze.
+
+\subsection{Focus}
+Joint attention is a well-studied and important task in Human-Robot Interaction. The goal of this task is to really challenge the teams to perform a hard HRI task.
+
+\subsection{Main Goal}
+The robot must interpret 3 point gestures of varying difficulty, must produce the same 3 point gestures, and must interpret the human's gaze gesture as well as generate one, and interpret the establishment of mutual gaze.
+
+\noindent\textbf{Reward:} 1000pts\\
+
+
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Setup
+%
+% %% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\subsection{Setup}
+\begin{enumerate}
+	\item \textbf{Surfaces:} The test area must have 3 surfaces with items arranged an equal distance apart on them.
+
+	\item \textbf{Items:} We need 3 different drinks, three different food items, and three different utensils arranged on each surface. The first surface should have items about 2 feet apart. The second should have them about 1.5 feet apart. The last should have them about 9 inches apart.
```

Comment:
ToDo: Shorten. Change to **S.I. UNITS**

A table featuring three groups of objects of the categories: *food*, *drinks*, and *utensils*.
I wouldn't say anything else.

Reply:
I think we should just get rid of the distance indications here. Seems unnecessary.